routes/users.js

```diff
@@ -21,6 +21,7 @@ router.get("/chaincode", authenticate, users.generateChaincode);
 router.get("/search", userValidator.validateUserQueryParams, users.filterUsers);
 router.patch("/remove-tokens", authenticate, authorizeRoles([SUPERUSER]), users.removeTokens);
 
+router.get("/githubInfo/:userId", users.userGithubInfo);
```

Comment:
please use a better api name githubInfo, seems bit odd. also we do not use camel case in URLs.

Reply:
ok, I will keep in mind from next time